packages/ts-sdk-apm/src/apm.ts

```diff
@@ -55,9 +40,14 @@ export default function createTelemetryMiddleware(
 
       // send `response_time` to APM platforms
       if (options?.customMetrics) {
-        options.customMetrics.newrelic && recordNewrelic(response_time)
-        options.customMetrics.datadog &&
-          recordDatadog(response_time, { env: 'dev' })
+        if (options.customMetrics.datadog) {
```

Comment:
Is this option `customMetrics` documented? I couldn't find it in the docs: https://docs.commercetools.com/sdk/observability/datadog#include-the-datadog-middleware-in-the-typescript-sdk

Also, in the docs link above it says that we are adding automatically 3 metrics for datadog (`commercetools_client_response<Success>`, `commercetools_client_response<Error>`, `commercetools_client_request_total`). However, I could not find a place where this is done. I could see we are adding with slightly different letter case the metric `Commercetools_Client_Response_Total`, but I can't find the other 2 metrics. Can you point to the place in the SDK?

Reply:
Thank you for noticing it!
I also didn't find them ;) and this code was already there. I am gonna add this to the [task](https://commercetools.atlassian.net/browse/DEVX-524)

Reply:
About the second question, it's not done in the SDK, but in the demo app itself:
please check the response.js file and in the app.js file.

Reply:
Yes, it makes sense, I am creating in parallel a branch in the docs repo to update these kind of information ;)

Reply:
Also because the official documentation is not everywhere updated with the new client

Reply:
@barbara79 can you link the branch in the docs here so I know you did it and can review it? 🙏

Reply:
I changed it with this part 

```
console.warn(
        'APM is missing or not a function. Proceeding without tracing.'
      )
      options.apm = () => {}
      
```
      please see the updated code and also the comment made by @ajimae

Reply:
I am gonna add the link soon.

Reply:
https://github.com/commercetools/commercetools-docs/blob/update-ts-docs/websites/sdk/src/content/observability/datadog.mdx

Reply:
I see it's not updated yet?